paper.tex

```diff
@@ -257,6 +257,32 @@ \subsection*{Population genetics model}
 over that interval, which can be a constant value or function defined by start
 and end sizes that must remain positive.
 
+\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+	I'm not making an attempt yet to revise that later content--I think other authors should
+	weigh in.}
+
+\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+
+Within a population, we implicitly assume that allele frequency dynamics can
+be described by the "Wright-Fisher" model.  Briefly, generations are non-overlapping (all parents
```

Comment:
latex quotes here like ``quote''. Probably don't need scare quotes on WF though

Reply:
I'll fix that.  One of the latex gotchas that always gets me. 🙄